roles/lnls-ans-role-repositories/templates/epicsdeb-repo-pref.j2

```diff
@@ -1,7 +1,3 @@
 Package: *
 Pin: origin epicsdeb.bnl.gov
-Pin-Priority: {{ epics_deb_main_conf_pin_prio }}
-
-Package: epics-perl
-Pin: origin epicsdeb.bnl.gov
-Pin-Priority: {{ epics_deb_epics_perl_conf_pin_prio }}
+Pin-Priority: {{ epics_deb_main_conf_pin_prio }}
```

Comment:
We shouldn't change anything in the "repositories" role I think. It doesn't install anything. It just sets the "Pin" priority so we can install Debian packages in earlier versions of Ubuntu.

Reply:
my github viewer complains about no newline at end of file here.

Reply:
@lerwys , ok, if you think epics-perl will ever be used, we can keep role-repositories as it is. done.

@fernandohds564 , please check if viewer stopped complaining.

Reply:
it ok now @xresende. thanks.